release-notes/opensearch-release-notes-1.3.11.md

```diff
@@ -0,0 +1,27 @@
+# OpenSearch and OpenSearch Dashboards 1.3.11 Release Notes
+
+## Release Details
+
+[OpenSearch and OpenSearch Dashboards 1.3.11](https://opensearch.org/versions/opensearch-1-3-11.html) includes the following bug fixes, infrastructure, enhancements and maintenance updates.
+
+OpenSearch [Release Notes](https://github.com/opensearch-project/OpenSearch/blob/main/release-notes/opensearch.release-notes-1.3.11.md).
+
+OpenSearch Dashboards [Release Notes](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/release-notes/opensearch-dashboards.release-notes-1.3.11.md).
+
+
+## ENHANCEMENTS
+
+## BUG FIXES
+
+## MAINTENANCE
+
+### Opensearch ML Commons
+* Increment version to 1.3.11-SNAPSHOT ([#983](https://github.com/opensearch-project/ml-commons/pull/983))
+* Manual CVE fix ([#1010](https://github.com/opensearch-project/ml-commons/pull/1010))
```

Comment:
Upgrade guava to 31.0.1 and checkstyle to 8.38.

Reply:
updated. thanks